modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/FieldsTable.js

```diff
@@ -20,9 +25,19 @@ import getFieldsFromSchema from './getFieldsFromSchema';
 
 function FieldsTable({portletNamespace}) {
 	const [fields, setFields] = useState([]);
+	const [selectedExportFileFormat, setSelectedExportFileFormat] = useState(
+		''
+	);
 	const [selectedFields, setSelectedFields] = useState([]);
 	const useTemplateMappingRef = useRef();
 
+	const isForbidden = (fieldType) => {
+		return (
+			FORBIDDEN_CSV_FIELD_TYPES.includes(fieldType) &&
```

Comment:
hey @jaimelr10 these fields are objects fields. I think it would be better to make sure we only apply this logic when the entity type is object. Instead of having an array of fields we could have a map:
```js
{
   'com.liferay.object....': [the forbidden fields...]
}
```

This way if another entity has fields with the same name, we will not break them.

Reply:
:point_up: